core/util.go

```diff
@@ -213,10 +213,34 @@ func IsAnyNilOrZero(vals ...interface{}) bool {
 		switch v := val.(type) {
 		case nil:
 			return true
+		case string:
+			if v == "" {
+				return true
+			}
+		case []string:
+			if len(v) == 0 {
+				return true
+			}
```

Comment:
Can the `[]string` and `[]byte` cases be combined? I legitimately don't know the answer to this question.

Reply:
They cannot, I tried that.